soc/nxp/mcx/nx4x/Kconfig.soc

```
@@ -0,0 +1,34 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_MCXNX4X
+	bool
+	select SOC_FAMILY_NXP_MCX
+
+config SOC_SERIES
+	default "nx4x" if SOC_SERIES_MCXNX4X
```

```suggestion
	default "mcxnx4x" if SOC_SERIES_MCXNX4X
```
And update soc.yml and board.yml

---

I have made this change